Description: Use the -g1 debugging level instead of -g when compiling WebKit
 to reduce the memory used and avoid build failures caused by lack of RAM.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/modules/web/src/main/native/Source/cmake/OptionsCommon.cmake
+++ b/modules/web/src/main/native/Source/cmake/OptionsCommon.cmake
@@ -43,6 +43,8 @@
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-expansion-to-defined")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-exceptions -fno-strict-aliasing -fno-rtti")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++1y")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g1")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g1")
 endif ()
 
 if (COMPILER_IS_CLANG AND CMAKE_GENERATOR STREQUAL "Ninja")
